### PR TITLE
Added CVE-2022-3676.json

### PR DESCRIPTION
**Pull request by mbarbero**

### CODE REVIEW

File: 2022/3xxx/CVE-2022-3676.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
```
Comment (cve-team) [inline 2022/3xxx/CVE-2022-3676.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/3xxx/CVE-2022-3676.json:1]: ('Expecting value: line 60 column 7 (char 1851)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @mbarbero,

Can you please resolve the merge conflict in this PR? Once resolved we can get this request processed. 

Thanks,
CVE Content Team

**Pull request merged**